File: master/Freigt-Easy/Freigt-Easy/Core/Entities/Order.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace Freigt_Easy.Core.Entities
{
    public class Order : BaseEntity
    {
        public string OrderUNId { get; set; }
        public int BlCount { get; set; }
        public string Commodity { get; set; }
        public string CommodityType { get; set; }
        public int Qty { get; set; }

        public float OriginFxValue { get; set; }
        public float DestFxValue { get; set; }
        public DateTime OrderDate { get; set; }
        public float TotalCharges { get; set; }

        public string RzOrderId{get;set;}
        public string RzPaymentId { get; set; }
        public string RzSignature{ get; set; }

        [ForeignKey("VesselChargeOrder")]
        public int VesselChargeId { get; set; }
        public VesselCharge VesselCharge { get; set; }
 
        public string TransactionStatus { get; set; }



        [NotMapped]
        public string IsCreditAllowed { get; set; }

        [NotMapped]
        public string RzrKey { get; set; }


    }
}

File: master/Freigt-Easy/Freigt-Easy/ClientApp/src/app/models/order.ts
import {VesselCharge} from './vessel-charge'
export class Order {
 
        public orderUNId: string;
        public blCount: number;
        public commodity: string;
        public commodityType: string;
        public qty: number;
        public originFxValue: number;
        public destFxValue: number;
        public orderDate: Date
        public totalCharges: number;
        public rzOrderId: string;
        public rzPaymentId: string;
        public rzSignature: string;
        public vesselChargeId: number;
        public transactionStatus: string;
        public isCreditAllowed: string;
        public id: number;
        public isActive  : boolean;
        public createdBy: string;
        public updatedBy: string;
        public createdAt: Date;
        public updatedAt: Date;
        public rzrKey : null

        public vesselCharge :VesselCharge;
      }
 
